README.md

```diff
@@ -700,7 +700,7 @@ SyntaxError: invalid syntax
 
 ---
 
-### ▶ Half triple-quoted strings
+### ▶ Half triple-quoted strings (which is wrong in orinial version)
```

Comment:
@moontree Hi, maybe you should fix the typo - `orinial` -> `original`.

Reply:
I have fix this, thanks!